src/codegen/enums.rs

```diff
@@ -236,6 +236,13 @@ fn generate_enum(
         )?;
     }
 
+    // Only inline from_glib / into_glib implementations if there are not many enums members
+    let maybe_inline = if members.len() <= 12 {
```

Comment:
12 is just a random number

Reply:
That's always 4 bytes :) The relevant metric here is the number of conditions in the match, which is equivalent to the number of enum variants.

Reply:
Oh I see.